### DISCUSSION TITLE
Save plotter as interactive 3D

### DISCUSSION THREAD

**Discussion by SteBarsanti** [pyvista/pyvista#1542] [Q&A] [answered]
Is there any way to save the output from the plotter as interactive 3D and not just as static image/screenshot?
I tried to use Jupyter and convert to .html but either it doesn't show or it's static (depend on the backend used when plotting). 
Thanks!

**Comment by akaszynski** [comment, marked as answer]
That's actually something I'm working on right now!  It's exporting using pythreejs and it's really great!  I'll have it out on the next release.

**Reply by SteBarsanti** [reply]
That's great! Thank you so much! I think it would be really useful for publications etc.

**Reply by anicusan** [reply]
Just a thumbs up for @akaszynski for working on interactive 3D exporting, I'm also looking forward to using it! It will be in HTML format I presume?

**Reply by akaszynski** [reply]
Will be in HTML as a standalone webpage in https://github.com/pyvista/pyvista/pull/1557
